DEVELOPERS.md

```diff
@@ -137,3 +137,14 @@ and `constant` (by source file) records.
 #### Rendering Coverage Info on api.codeclimate.com
 
 TODO
+
+## Releasing
+
+To release a new version run:
+
+```
+make release VERSION=<version number>
```

Comment:
Sorry, I could've been clearer before :) the typical release steps would be:

1. Update `VERSION` in `Makefile` and commit
1. `git tag -s -m v{VERSION} v{VERSION}` and push and update release notes
1. Run `make release` to build and publish new binaries

I'm now realizing we could put the `git tag` bit in the `make release` step.